docs/settings.md

```diff
@@ -30,6 +30,7 @@ equivalent keyword arguments, eg. `uvicorn.run("example:app", port=5000, reload=
 * `--log-config <path>` - Logging configuration file.
 * `--log-level <str>` - Set the log level. **Options:** *'critical', 'error', 'warning', 'info', 'debug', 'trace'.* **Default:** *'info'*.
 * `--no-access-log` - Disable access log only, without changing log level.
+* `--use-colors` - Enable colorized formatting of the log records.
```

Comment:
We should change this doc around so that:

* We include both the "on" and "off" flag variants.
* It's clear that we'll autodetect if the flag isn't used.

Reply:
changed, let me know if this is better